android/KMAPro/kMAPro/src/main/java/com/keyman/android/SystemKeyboard.java

```diff
@@ -69,6 +69,9 @@ public void onCreate() {
     KMManager.SpacebarText spacebarText = KMManager.SpacebarText.fromString(prefs.getString(KeymanSettingsActivity.spacebarTextKey, KMManager.SpacebarText.LANGUAGE_KEYBOARD.toString()));
     KMManager.setSpacebarText(spacebarText);
 
+    boolean mayVibrateWhenTyping = prefs.getBoolean(KeymanSettingsActivity.vibrateWhenTypingKey, false);
+    KMManager.setMayVibrateWhenTyping(mayVibrateWhenTyping);
```

Comment:
It'd be good if we used a consistent name for the variable and the setting throughout, e.g. `hapticFeedback` and `hapticFeedbackKey`.